### ISSUE TITLE
`path-for` ignores trailing part after path variable

### ISSUE THREAD

**Issue by ulsa** [metosin/compojure-api#150]
I have this:

```
(POST* "/companies/:company-id/refresh" []
  :name :refresh-company
  ...
```

and refer to it like this:

```
(found (path-for :refresh-company {:company-id id}))
```

but get in `Location` this:

```
http://127.0.0.1:3000/api/companies/4/
```

ie, missing the trailing `refresh`. Expected the path to be:

```
http://127.0.0.1:3000/api/companies/4/refresh
```

**Comment by ikitommi** [comment]
will check this out.

**Comment by ikitommi** [comment]
should be fixed in `[metosin/compojure-api "0.22.2"]`.
